deploy-as-code/helm/environments/ci.yaml

```diff
@@ -153,7 +153,13 @@ jenkins:
     - name: unified-urban-dev
       acl: [egovernments*unified-urban-dev]
     - name: punjab-fsm
-      acl: [egovernments*punjab-fsm]  
+      acl: [egovernments*punjab-fsm]
+    - name: moz-health-qa
```

Comment:
please add in moz-health group common users can't deploy in these clusters

Reply:
Why same group is getting used for prd and all other envs?